crates/cxx-qt-gen/test_inputs/signals.rs

```diff
@@ -36,6 +36,12 @@ mod ffi {
     impl qobject::MyObject {
         #[qinvokable]
         pub fn invokable(self: Pin<&mut Self>) {
+            self.as_mut().on_data_changed(
+                |_, _, _, _| {
```

Comment:
This should be five parameters here, as you always get the `self` qobject as the first parameter, right?
Maybe also add names here, e.g.: 
```suggestion
                |_sender,_first, _second, _third, _fourth| {
```

Reply:
Oops, can you tell we don't actually build these examples and they are just for syntax checking :-)

Reply:
Done